snooty/util.py

```diff
@@ -114,13 +119,22 @@ def get_files(
         # Preserve both the actual resolved path and the directory name
         dirs_set = dict(((base_resolved.joinpath(d).resolve(), d) for d in dirs))
 
+        nested_set = set()
+        for d_path, d_name in dirs_set.items():
+            dir_snooty_path = d_path / SNOOTY_TOML
```

Comment:
TIL about truediv

Reply:
I actually didn't know about truediv until I googled it just now! Haha. I just saw that we were joining paths elsewhere in the codebase with a forward slash